app/modules/site_settings/helpers.py

```diff
@@ -342,6 +388,27 @@ def is_valid_value_for_class(cls, cls_name, cfd_id, value):
             return False
         return True
 
+    # DEX-1337 PATCH op=remove path=site.custom.customFields.CLASS/GUID must be supported
+    #   this also will be called for (currently unused) PATCH op=remove path=site.custom.customFieldCategories fwiw (see DEX-1362)
+    #   force_removal=True means go ahead and blow away all the data that exists.  ouch!
+    #
+    #   note: this *intentionally* disallows straight-up reseting entire customFields.CLASS object.  youre welcome.
+    @classmethod
+    def patch_remove(cls, key, force_removal=False):
+        import re
+
+        m = re.search(r'site.custom.customFields.(\w+)/([\w\-]+)', key)
```

Comment:
please no single character variable names. It really does make reading code much more difficult. 
I'm now guessing what m means

Reply:
i am not sure if `match_obj` will have you guessing any less, but i have renamed it as such. haha.   its a ... thing that `search()` returns with a structure specific to regex results.

Reply:
[match object](https://docs.python.org/3/library/re.html#match-objects), fwiw.